sprint_metrics.py

```diff
@@ -143,30 +144,29 @@ def getSprintMetrics(sprint_report):
             continue
 
         try:
-            issue_points = int(completed["currentEstimateStatistic"]["statFieldValue"]["value"])
+            issue_points_original = int(completed["estimateStatistic"]["statFieldValue"]["value"])
         except:
-            issue_points = 0
+            issue_points_original = 0
 
         try:
-            issue_points_original = int(completed["estimateStatistic"]["statFieldValue"]["value"])
+            issue_points = int(completed["currentEstimateStatistic"]["statFieldValue"]["value"])
         except:
-            issue_points_original = 0
+            issue_points = 0
 
         points["completed"] += issue_points
         items["completed"] += 1
 
         unplanned = False
         if completed["key"] in sprint_report["contents"]["issueKeysAddedDuringSprint"].keys():
             unplanned = True
-            points["unplanned_completed"] += issue_points
+            points["unplanned_completed"] += issue_points_original
             items["unplanned_completed"] += 1
         else:
-
             points["committed"] += issue_points_original
             items["committed"] += 1
-            points["planned_completed"] += issue_points_original
+            points["planned_completed"] += issue_points
             items["planned_completed"] += 1
-            if issue_points_original != issue_points:
+            if issue_points_original > issue_points:
```

Comment:
We could end up with negative unplanned points the way this was written. I think this approach makes more sense (only changing the unplanned points for increases in estimate)